modules/imgproc/include/opencv2/imgproc.hpp

```diff
@@ -1169,7 +1169,10 @@ class CV_EXPORTS_W Subdiv2D
     void splice( int edgeA, int edgeB );
     int connectEdges( int edgeA, int edgeB );
     void swapEdges( int edge );
-    int isRightOf(Point2f pt, int edge) const;
```

Comment:
the patch changes external API. Is it really necessary to do it? Can CCWEx, RightOfEx and InCircleEx be made internal functions?

Reply:
Thanks for the advice. It was possible to make the functions to be private members, however, there is one potential issue with meta points being visible to subclasses.